test/php/regression/Bug3239Test.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/* Icinga Web 2 | (c) 2013 Icinga Development Team | GPLv2+ */
```

Comment:
Please always use the current year in newly created files.

Reply:
Whops! The copy/paste was strong with me :) fixed